tests/cpp_api_tests/rocAL_audio_unittests/rocAL_audio_unittests.cpp

```diff
@@ -0,0 +1,250 @@
+            // RocalTensor begin = non_silent_region->at(0);
+        }
+        break;
+        case 4:
+        {
+            std::cout<<"\n Mel Filter Bank";
+            RocalTensorLayout tensorLayout; // = RocalTensorLayout::None;
+            RocalTensorOutputType tensorOutputType = RocalTensorOutputType::ROCAL_FP32;
+            std::vector<float> window_fn{};
+            RocalTensor temp_output = rocalSpectrogram(handle, input1, tensorOutputType, false, window_fn, true, true, RocalSpectrogramLayout(0), 2, 512, 512, 256);
+            float sampleRate = 16000;
+            float minFreq = 0.0;
+            float maxFreq = sampleRate / 2;
+            RocalMelScaleFormula melFormula = RocalMelScaleFormula::SLANEY;
+            int numFilter = 128;
+            bool normalize = true;
+
+            output = rocalMelFilterBank(handle, temp_output, true, maxFreq, minFreq, melFormula, numFilter, normalize, sampleRate);
+        }
+        break;
+        case 5:
+        {
+            RocalTensorLayout tensorLayout; // = RocalTensorLayout::None;
+            RocalTensorOutputType tensorOutputType = RocalTensorOutputType::ROCAL_FP32;
+            const size_t num_values = 3;
+            std::pair <RocalTensor,RocalTensor>  non_silent_region_output;
+            non_silent_region_output = rocalNonSilentRegion(handle, input1, false, -60, 0.0, -1, 3);
+            output = rocalSlice(handle, input1, tensorOutputType, true, non_silent_region_output.first, non_silent_region_output.second, {0.3f});
+        }
+        break;
+        case 6:
+        {
+            std::cout<<"\n Normalize";
+            RocalTensorLayout tensorLayout;
+            RocalTensorOutputType tensorOutputType = RocalTensorOutputType::ROCAL_FP32;
+            output = rocalNormalize(handle, input1, tensorOutputType, true, false, {1});
+        }
+        break;
+        case 7:
+        {
+            std::cout<<"\nPad";
+            RocalTensorLayout tensorLayout;
+            RocalTensorOutputType tensorOutputType = RocalTensorOutputType::ROCAL_FP32;
+            output = rocalPad(handle, input1, tensorOutputType, true, 4.0f);
+        }
+        break;
+
+        default:
+        {
+            std::cout << "Not a valid pipeline type ! Exiting!\n";
+            return -1;
+        }
+
+    }
+    */
+    rocalVerify(handle);
+    if (rocalGetStatus(handle) != ROCAL_OK) {
+        std::cout << "Could not verify the augmentation graph " << rocalGetErrorMessage(handle);
+        return -1;
+    }
+
+    /*>>>>>>>>>>>>>>>>>>> Diplay using OpenCV <<<<<<<<<<<<<<<<<*/
+    cv::Mat mat_output, mat_input, mat_color;
+    int iteration = 0;
+    RocalTensorList output_tensor_list;
+
+    while (rocalGetRemainingImages(handle) >= static_cast<size_t>(inputBatchSize)) {
+        std::cout << "\n rocalGetRemainingImages:: " << rocalGetRemainingImages(handle) << "\t inputBatchsize:: " << inputBatchSize;
+        std::cout << "\n iteration:: " << iteration;
```

Comment:
Do we need this print statement and iteration variable?

Reply:
Just to show which iteration of an epoch it belongs to - we need the statement